components/raftstore/src/store/async_io.rs

```diff
@@ -0,0 +1,949 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+
+// TODO: move it
+#![allow(dead_code)]
```

Comment:
How about splitting the actual implementation and tests into two files?

Reply:
I don't get it. The tests in this file are unit tests.

Reply:
Yes, but they can still be split into other files. https://github.com/tikv/tikv/blob/master/src/storage/txn/actions/tests.rs is an example. The intention is to avoid too long files.